src/views/contact-us/l10n.json

```diff
@@ -0,0 +1,16 @@
+{
+    "contactUs.title":"Contact Us",
+    "contactUs.intro":"You can find answers to most questions about Scratch on our {faqLink} page.",
+    "contactUs.faqLinkText":"Frequently Asked Questions",
+    "contactUs.forumsInfo":"You can also look through the Scratch {forumsLink}.",
+    "contactUs.forumsLinkText":"Discussion forums",
+    "contactUs.forumsAsk":"If you can't find an answer, post a new topic on the {forumsLink} (see links below). There are many experienced Scratchers who can help with the topics below and more:",
+    "contactUs.questionsForum":"Questions About Scratch",
+    "contactUs.scriptsForum":"Help with Scripts",
+	"contactUs.bugsForum":"Bugs and Glitches",
+    "contactUs.formIntro":"If you still need to contact us, please fill out the form below with as much detail as you can. If you have any screenshots, attachments or links that help to explain your problem, please include them. We get a lot of mail, so we may not be able to respond to your message.",
+    "contactUs.feedbackForm":"Feedback Form",
+    "contactUs.findHelp":"Where to find help:",
+    "contactUs.knowledgeBase":"Knowledge Base",
```

Comment:
This is unused; will it be in the future?

Reply:
Yes, these are for some follow on work.